### PR TITLE
feat: mock nfts and txs hiro requests

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out Leather build e52e0d9 — [Extension build](https://github.com/leather-io/extension/actions/runs/10512923672), [Test report](https://leather-io.github.io/playwright-reports/feat-mock-hiro), [Storybook](https://feat-mock-hiro--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=feat-mock-hiro)

This pr adds mocking of hiro txs, bns names and nfts requests in tests

### CODE REVIEW

File: tests/mocks/mock-stacks-txs.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import type { Page } from '@playwright/test';
+
+import { TEST_ACCOUNT_1_STX_ADDRESS } from './constants';
+
+export async function mockMainnetTestAccountStacksTxsRequests(page: Page) {
+  await Promise.all([
+    page.route(
+      `**/api.hiro.so/extended/v1/address/${TEST_ACCOUNT_1_STX_ADDRESS}/transactions_with_transfers?limit=50`,
```
Comment (pete-watters) [inline tests/mocks/mock-stacks-txs.ts:8]: Maybe we can use a constant / types for the URL? 

`api.hiro.so/extended/v1/`

**Review by pete-watters — APPROVED** [review]

**Comment by markmhendrickson** [comment]
Is this issue needed for linking? https://github.com/leather-io/issues/issues/193

**Comment by alter-eggo** [comment]
@markmhendrickson I'm continue working on the task, there will be one more pr at least

File: tests/mocks/mock-stacks-txs.ts (hunk 1 above)
Comment (alter-eggo) [reply tests/mocks/mock-stacks-txs.ts:8]: yeah, thanks, I'll add in next pr

**Pull request merged**